---a/net/minecraft/world/gen/feature/WorldGenMegaPineTree.java
+++b/net/minecraft/world/gen/feature/WorldGenMegaPineTree.java
@@ -1,6 +1,7 @@
 package net.minecraft.world.gen.feature;
 
 import java.util.Random;
+
 import net.minecraft.block.Block;
 import net.minecraft.block.BlockDirt;
 import net.minecraft.block.BlockPlanks;
@@ -10,58 +10,48 @@
 import net.minecraft.util.MathHelper;
 import net.minecraft.world.World;
 
-public class WorldGenMegaPineTree extends WorldGenHugeTrees
-{
+public class WorldGenMegaPineTree extends WorldGenHugeTrees {
+
     private boolean field_150542_e;
     private static final String __OBFID = "CL_00000421";
 
-    public WorldGenMegaPineTree(boolean p_i45457_1_, boolean p_i45457_2_)
-    {
+    public WorldGenMegaPineTree(boolean p_i45457_1_, boolean p_i45457_2_) {
         super(p_i45457_1_, 13, 15, BlockPlanks.EnumType.SPRUCE.getMetadata(), BlockPlanks.EnumType.SPRUCE.getMetadata());
         this.field_150542_e = p_i45457_2_;
     }
 
-    public boolean generate(World worldIn, Random p_180709_2_, BlockPos p_180709_3_)
-    {
+    public boolean generate(World worldIn, Random p_180709_2_, BlockPos p_180709_3_) {
         int i = this.func_150533_a(p_180709_2_);
 
-        if (!this.func_175929_a(worldIn, p_180709_2_, p_180709_3_, i))
-        {
+        if (!this.func_175929_a(worldIn, p_180709_2_, p_180709_3_, i)) {
             return false;
-        }
-        else
-        {
+        } else {
             this.func_150541_c(worldIn, p_180709_3_.getX(), p_180709_3_.getZ(), p_180709_3_.getY() + i, 0, p_180709_2_);
 
-            for (int j = 0; j < i; ++j)
-            {
+            for (int j = 0; j < i; ++j) {
                 Block block = worldIn.getBlockState(p_180709_3_.up(j)).getBlock();
 
-                if (block.getMaterial() == Material.air || block.getMaterial() == Material.leaves)
-                {
+                if (block.getMaterial() == Material.air || block.getMaterial() == Material.leaves) {
                     this.func_175905_a(worldIn, p_180709_3_.up(j), Blocks.log, this.woodMetadata);
                 }
 
-                if (j < i - 1)
-                {
+                if (j < i - 1) {
                     block = worldIn.getBlockState(p_180709_3_.add(1, j, 0)).getBlock();
 
-                    if (block.getMaterial() == Material.air || block.getMaterial() == Material.leaves)
-                    {
+                    if (block.getMaterial() == Material.air || block.getMaterial() == Material.leaves) {
                         this.func_175905_a(worldIn, p_180709_3_.add(1, j, 0), Blocks.log, this.woodMetadata);
                     }
 
                     block = worldIn.getBlockState(p_180709_3_.add(1, j, 1)).getBlock();
 
-                    if (block.getMaterial() == Material.air || block.getMaterial() == Material.leaves)
-                    {
+                    if (block.getMaterial() == Material.air || block.getMaterial() == Material.leaves) {
                         this.func_175905_a(worldIn, p_180709_3_.add(1, j, 1), Blocks.log, this.woodMetadata);
                     }
 
                     block = worldIn.getBlockState(p_180709_3_.add(0, j, 1)).getBlock();
 
-                    if (block.getMaterial() == Material.air || block.getMaterial() == Material.leaves)
-                    {
+                    if (block.getMaterial() == Material.air || block.getMaterial() == Material.leaves) {
                         this.func_175905_a(worldIn, p_180709_3_.add(0, j, 1), Blocks.log, this.woodMetadata);
                     }
                 }
@@ -71,72 +71,58 @@
         }
     }
 
-    private void func_150541_c(World worldIn, int p_150541_2_, int p_150541_3_, int p_150541_4_, int p_150541_5_, Random p_150541_6_)
-    {
+    private void func_150541_c(World worldIn, int p_150541_2_, int p_150541_3_, int p_150541_4_, int p_150541_5_, Random p_150541_6_) {
         int i1 = p_150541_6_.nextInt(5) + (this.field_150542_e ? this.baseHeight : 3);
         int j1 = 0;
 
-        for (int k1 = p_150541_4_ - i1; k1 <= p_150541_4_; ++k1)
-        {
+        for (int k1 = p_150541_4_ - i1; k1 <= p_150541_4_; ++k1) {
             int l1 = p_150541_4_ - k1;
-            int i2 = p_150541_5_ + MathHelper.floor_float((float)l1 / (float)i1 * 3.5F);
+            int i2 = p_150541_5_ + MathHelper.floor_float((float) l1 / (float) i1 * 3.5F);
             this.func_175925_a(worldIn, new BlockPos(p_150541_2_, k1, p_150541_3_), i2 + (l1 > 0 && i2 == j1 && (k1 & 1) == 0 ? 1 : 0));
             j1 = i2;
         }
     }
 
-    public void func_180711_a(World worldIn, Random p_180711_2_, BlockPos p_180711_3_)
-    {
+    public void func_180711_a(World worldIn, Random p_180711_2_, BlockPos p_180711_3_) {
         this.func_175933_b(worldIn, p_180711_3_.west().north());
         this.func_175933_b(worldIn, p_180711_3_.east(2).north());
         this.func_175933_b(worldIn, p_180711_3_.west().south(2));
         this.func_175933_b(worldIn, p_180711_3_.east(2).south(2));
 
-        for (int i = 0; i < 5; ++i)
-        {
+        for (int i = 0; i < 5; ++i) {
             int j = p_180711_2_.nextInt(64);
             int k = j % 8;
             int l = j / 8;
 
-            if (k == 0 || k == 7 || l == 0 || l == 7)
-            {
+            if (k == 0 || k == 7 || l == 0 || l == 7) {
                 this.func_175933_b(worldIn, p_180711_3_.add(-3 + k, 0, -3 + l));
             }
         }
     }
 
-    private void func_175933_b(World worldIn, BlockPos p_175933_2_)
-    {
-        for (int i = -2; i <= 2; ++i)
-        {
-            for (int j = -2; j <= 2; ++j)
-            {
-                if (Math.abs(i) != 2 || Math.abs(j) != 2)
-                {
+    private void func_175933_b(World worldIn, BlockPos p_175933_2_) {
+        for (int i = -2; i <= 2; ++i) {
+            for (int j = -2; j <= 2; ++j) {
+                if (Math.abs(i) != 2 || Math.abs(j) != 2) {
                     this.func_175934_c(worldIn, p_175933_2_.add(i, 0, j));
                 }
             }
         }
     }
 
-    private void func_175934_c(World worldIn, BlockPos p_175934_2_)
-    {
-        for (int i = 2; i >= -3; --i)
-        {
+    private void func_175934_c(World worldIn, BlockPos p_175934_2_) {
+        for (int i = 2; i >= -3; --i) {
             BlockPos blockpos1 = p_175934_2_.up(i);
             Block block = worldIn.getBlockState(blockpos1).getBlock();
 
-            if (block == Blocks.grass || block == Blocks.dirt)
-            {
+            if (block == Blocks.grass || block == Blocks.dirt) {
                 this.func_175905_a(worldIn, blockpos1, Blocks.dirt, BlockDirt.DirtType.PODZOL.getMetadata());
                 break;
             }
 
-            if (block.getMaterial() != Material.air && i < 0)
-            {
+            if (block.getMaterial() != Material.air && i < 0) {
                 break;
             }
         }
     }
-}
+}
